src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -175,6 +179,7 @@ def user_login(
             object_id=config.security.oauth_root_client_id,
             config=config,
             scopes=config.security.root_user_scopes,
+            roles=config.security.root_user_roles,
```

Comment:
On login, adding both the root user scopes and root user roles to the ClientDetail for the root user